### ISSUE TITLE
When you expand dropdown in high contrast mode, at a time two options(items) are selected.

### ISSUE THREAD

**Issue by Harkesh030679** [microsoft/fluentui#12430]
When you expand dropdown in high contrast mode, at a time two options(items) are selected.

### Environment Information

OS: Win 10
Browser:  Edge

### Describe the issue:

When you expand dropdown in high contrast mode, at a time two options(items) are selected.
Please see below images in both high contrast mode

![1715925_Black_Fixed_foroffice](https://user-images.githubusercontent.com/62233836/77620090-9b854480-6f5f-11ea-957e-1579f1fda124.png)

![1715925_White_Fixed_foroffice](https://user-images.githubusercontent.com/62233836/77620131-afc94180-6f5f-11ea-90c7-dea0f10319ba.png)

### Please provide a reproduction of the issue in a codepen:

Repro at below link in high contrast mode
https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown

#### Actual behavior:

When you expand dropdown in high contrast mode, at a time two options(items) are selected.
#### Expected behavior:

When you expand dropdown in high contrast mode, at a time only one option(item) are selected.

**Comment by aneeshack4** [comment]
@Harkesh030679 Thanks for reporting this! I confirm repro. I actually see this without high contrast too:
![image](https://user-images.githubusercontent.com/4161791/77706883-97721a80-6f91-11ea-91fd-60fae4c85d29.png)
The first selected option has the hover background even when keyboarding/arrowing down to other options. I think this might be a possible regression of #11218 - @xugao what do you think?
